Lib/fontTools/ttLib/tables/otConverters.py

```diff
@@ -338,6 +338,14 @@ def readArray(self, reader, font, tableDict, count):
 		return l
 	def read(self, reader, font, tableDict):
 		return font.getGlyphName(reader.readValue(self.typecode, self.staticSize))
+	def writeArray(self, writer, font, tableDict, values):
+		glyphMap = font.getReverseGlyphMap()
+		try:
+			values = [glyphMap[glyphname] for glyphname in values]
+		except KeyError:
+			# Slower, but will not throw a KeyError on an out-of-range glyph name.
+			values = [font.getGlyphID(glyphname) for glyphname in values]
+		writer.writeArray(self.typecode, values)
```

Comment:
is this change related to https://github.com/fonttools/fonttools/issues/1536#issuecomment-472581569 ?

Reply:
Is what I proposed there as well. I didn't remember that. Just came across it when profiling.